src/CTA.WebForms2Blazor/ControlConverters/ControlConverter.cs

```diff
@@ -0,0 +1,18 @@
+﻿using HtmlAgilityPack;
+
+namespace CTA.WebForms2Blazor.ControlConverters
+{
+    public abstract class ControlConverter
+    {
+        protected ControlConverter()
+        {
+            //Constructor might not be needed
```

Comment:
It's probably true that we probably don't need the constructor, having it here in case we need it is probably fine but removing it wouldn't hurt either, we can just add it back later if needed